output_3.2.txt

```diff
@@ -0,0 +1,21 @@
+TASK: 3.2
```

Comment:
+ in this task there should be only one query
+ add query results
+ all there queries are not correct (and, I guess, have different results)